.buildkite/pipelines/quality-gates/pipeline.tests-qa.yaml

```diff
@@ -46,5 +38,7 @@ steps:
   - label: ":judge::seedling: Trigger Manual Tests Phase"
     command: "make -C /agent trigger-manual-verification-phase"
     if: build.branch == "main"
+    env:
+      TEAM_CHANNEL: "#kibana-mission-control"
```

Comment:
```suggestion
```
you don't need this. Its a variable that is already defined as part of the common entry for all environment quality gates in `pipeline.kibana-tests.yaml`:
https://github.com/elastic/kibana/blob/3ad379ad3e07248709e9110ba87fc831e09d162b/.buildkite/pipelines/quality-gates/pipeline.kibana-tests.yaml#L14

Reply:
oh, thanks! so it wasn't necessary in the block I removed either, I suppose